src/Arcus.Security.Tests.Integration/Logging/XunitTestLogger.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit.Abstractions;
+using Microsoft.Extensions.Logging;
+using GuardNet;
+
+namespace Arcus.Security.Tests.Integration.Logging
+{
+    public class XunitTestLogger : ILogger
```

Comment:
Looks like copy paste from event grid, is that correct?

If so, please add an issue https://github.com/arcus-azure/arcus to create https://github.com/arcus-azure/arcus.core with something like this.

Reply:
true - was copy/paste, as we might need this in multiple projects, it would be good to have these testing things centralized.  you mention Arcus.Core, but would it be valuable to have something like Arcus.Testing as well?  I'll leave it up to you to create the issue where you deem it fit 
Cheers

Reply:
That would also be a good one, please create an issue on Arcus (general one) to keep track of this and we can move the conversation there - Will forget about it due to holidays :D

Reply:
Created https://github.com/arcus-azure/arcus/issues/19 for this